scripts/build_sc_machine.sh

```diff
@@ -26,25 +38,28 @@ while [ "$1" != "" ]; do
 		"-t"|"--tests" )
 			build_tests=1
 			;;
-    "-r" | "--release" )
-      release_mode=1
+    "-r"|"--release" )
+      build_release=1
       ;;
-	  "-h"|"--help")
-      echo "Usage: $(basename "$0") [OPTION]..."
-      echo
-      echo "Options:"
-      echo "  -f, --force     full rebuild with the deleting of the $(relative "${BIN_PATH}") and $(relative "${BUILD_PATH}") folders"
-      echo "  -t, --tests     build tests"
-      echo "  -r, --release   release mode"
-      echo "  -h, --help      display this help and exit"
+    "--add-options-info" )
+      shift 1
+      additional_options+=(${1})
```

Comment:
I don't see this variable being used anywhere. Moreover, it's not documented in the help menu of the script